submissions/maxim-zabolotny/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,122 @@
+                </a>
+                <a href="#" class="icon_card">
+                  <img src="img/contacts.png" alt="Contacts">
+                  Contacts
+                </a>
+                <a href="#" class="icon_card">
+                  <img src="img/tanslation.png" alt="Translate">
+                  Translate
+                </a>
+                <a href="#" class="icon_card">
+                  <img src="img/videos.png" alt="Videos">
+                  Videos
+                </a>
+                <a href="#" class="icon_card">
+                  <img src="img/search.png" alt="Search">
+                  Search
+                </a>
+                <a href="#" class="icon_card">
+                  <img src="img/photos.png" alt="Photo">
+                  Photo
+                </a>
+                <a href="#" class="icon_card">
+                  <img src="img/apps.png" alt="Apps">
+                  Apps
+                </a>
+                <a href="#" class="icon_card">
+                  <img src="img/documents.png" alt="Document">
+                  Document
+                </a>
+              </div>
+              <div class="first_page__footer">
+                <input type="checkbox" class="more_apps__checkbox" id="m_a_checkbox">
+                <label class="more_apps__label" for="m_a_checkbox">More</label>
+                <div class="second_page">
+                  <a href="#" class="icon_card">
+                    <img src="img/chat.png" alt="Chat">
+                    Chat
+                  </a>
+                  <a href="#" class="icon_card">
+                    <img src="img/classroom.png" alt="Classroom">
+                    Classroom
+                  </a>
+                  <a href="#" class="icon_card">
+                    <img src="img/collection.png" alt="Collection">
+                    Collection
+                  </a>
+                  <a href="#" class="icon_card">
+                    <img src="img/mars.png" alt="Mars">
+                    Mars
+                  </a>
+                  <a href="#" class="icon_card">
+                    <img src="img/blogs.png" alt="Blogs">
+                    Blogs
+                  </a>
+                  <a href="#" class="icon_card">
+                    <img src="img/apps.png" alt="Ads">
+                    Ads
+                  </a>
+                  <div class="second_page__footer">
+                    <a href="https://kottans.org/">Even more from Kottans</a>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </li>
+        <li class="user_menu__element">
+          <a href="#"><img src="img/bell.png" class="bell_img" alt="bell"></a>
+        </li>
+        <li class="user_menu__element">
+          <a href="https://github.com/maxim-zabolotny/kottans-frontend"><img class="user_menu__user_photo" src="img/elon musk.jpeg" alt="user_photo"></a>
+        </li>
```

Comment:
Check your website using validator https://validator.w3.org/

Reply:
thanks for the help